docs/docsite/rst/conf.py

```diff
@@ -386,3 +392,22 @@
 ]
 linkcheck_workers = 25
 # linkcheck_anchors = False
+
+# Generate redirects for pages when building on Read The Docs
+def setup(app: Sphinx) -> dict[str, str]:
+
+    if tags.has('redirects'):
+
+        redirects = {}
+        redirects_config_path = DOCS_ROOT_DIR.parent / "ansible_redirects.toml"
```

Comment:
Have we decided if it'd make sense to consolidate all the TOML config in subfolder?
```suggestion
        redirects_config_path = DOCS_ROOT_DIR.parent / "declarative-configs" / "ansible_redirects.toml"
```

Reply:
Let's do it.